.aws/src/events-schema/prospectEvents.ts

```diff
@@ -0,0 +1,2 @@
+// TODO: do we need this file? ./userEvents.ts is imported in ../main.ts, but is never
```

Comment:
this folder will have the schema bindings generated by the schema-registry console..the dev has to manually generate it and copy-paste it here 

the consuming application can copy this code into their repository, e.g https://github.com/Pocket/account-data-deleter/blob/main/lambda/events/schemas/accountDeleteEvent.ts